OWDTestToolkit/DOM/Music.py

```diff
@@ -2,8 +2,8 @@
 
 audio = ('id', 'player-audio')
 done_button = ("xpath", "//*[@id='title-done']")
-song_by_title = ("xpath", "//span[@class='list-main-title'][text()='{}']")
-first_song = ("xpath", '//*[text()="MP4"]')
+song_by_title = ("xpath", '//a[@data-key-range="{}"]')
+first_song = ("xpath", '//a[@data-option="title"]') # Must be xpath
```

Comment:
Si dice must be a xpath, será por algo :). No seré yo quien contradiga a un todopoderoso cometnario